src/avatar.scss

```diff
@@ -0,0 +1,158 @@
+@import "./settings";
```

Comment:
we should not depened in `./settings`, instead we should use `@import "./new-settings";`

Reply:
Also we should consider using it, because there are no such things needed from `settings` in this file